cylc/flow/scheduler.py

```diff
@@ -1005,11 +1006,20 @@ def process_queued_task_messages(self) -> None:
         # Remaining unprocessed messages have no corresponding task proxy.
         # For example, if I manually set a running task to succeeded, the
         # proxy can be removed, but the orphaned job still sends messages.
-        for tms in messages.values():
-            warn = "Undeliverable task messages received and ignored:"
-            for _, msg in tms:
-                warn += f'\n  {msg.job_id}: {msg.severity} - "{msg.message}"'
-            LOG.warning(warn)
+        warn = ""
+        for tm in unprocessed_messages:
+            job_tokens = self.tokens.duplicate(
+                Tokens(tm.job_id, relative=True)
+            )
+            tdef = self.config.get_taskdef(job_tokens['task'])
+            if not self.task_events_mgr.process_job_message(
+                job_tokens, tdef, tm.message, tm.event_time
+            ):
```

Comment:
An undeliverable task message (no task proxy in the pool) with no job submit number brings the scheduler down here, on this branch, but not on master.

 `cylc message <workflow> 1/a "hello"`  (as opposed to `1/a/01`) with no `1/a` in the pool.

Weirdly, with `1/a` in the pool, this is fine - the submit number seems to default to 01.

That should not happen naturally, but maybe we should protect against it?